test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -172,7 +172,7 @@ public void KmeansOnnxConversionTest()
             var model = pipeline.Fit(data);
             var transformedData = model.Transform(data);
 
-            var onnxModel = mlContext.Model.ConvertToOnnxProtobuf(model, data);
+            var onnxModel = mlContext.Model.ConvertToOnnxProtobuf(model, data, 11);
```

Comment:
Will the opset for these tests always be 11? If it's likely to change, would it make sense to make a variable with the current opset?

Reply:
We will change this implementation, just ignore it. The initial thought was to set the default opset version to 9 and explicitly set new version of opset if the transformer requires newer opset version